capstone/about.html

```diff
@@ -0,0 +1,130 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <title>Rock & Roll Hall of Fame</title>
+
+</head>
+<body>
+  <header>
+    <div class="topBar">
+      <i class="fa-brands fa-facebook-f"></i>
+      <i class="fa-brands fa-twitter"></i>
+      <a href="#">English</a>
+      <a href="#">My page</a>
+      <a href="#">Logout</a>
+    </div>
+    <nav>
+      <div class="navbar">
+        <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+        <div class="nav-items">
+          <a class="aboutPage" href="about.html">About</a>
+          <a href="home.html#program">Program</a>
+          <a href="home.html#join">Join</a>
+          <a href="about.html#sponsor">Sponsor</a>
+          <a href="about.html#news">News</a>
+          <a class="rockHall" href="about.html#rockhall">Rock Hall</a>
+        </div>
+      </div>
+    </nav>
+    <div class="menu">
+      <a class="menu-button open" href="#"><i class="fa-solid fa-bars"></i></a>
+      <a class="menu-button close" href="#"><i class="fa-solid fa-xmark"></i></a>
+      <div hidden class="menu-content">
+        <a class="logo" href="home.html" ><img src="images/logo.svg"></a>
+        <div class="menu-items">
+          <a class="aboutPage" href="about.html">About</a>
+          <a href="home.html#program">Program</a>
+          <a href="home.html#join">Join</a>
+          <a href="about.html#sponsor">Sponsor</a>
+          <a href="about.html#news">News</a>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="main">
+      <div class="homeContent about">
+        <h2 class="greet">"Hello! Rock & Roll Fans"</h2>
+        <div class="holder">
+          <h1 class="title">ROCK & ROLL HALL OF FAME 2023</h1>
+          <div class="underLine"></div>
+        </div>
+        <p class="description">
+          <span>Rock and Roll Hall of Fame and Museum, museum and hall of fame in Cleveland that celebrates the history and cultural significance of rock music and honours the contributions of those who have played an important role in the music's creation and dissemination.</span>
+          <span class="deskAboutDesc">To be eligible for nomination, an individual artist or band must have released its first commercial recording at least 25 years prior to the year of nomination. Eight out of 14 of the Nominees are on the ballot for the first time, including Sheryl Crow, Missy Elliott, Joy Division/New Order, Cyndi Lauper, George Michael, Willie Nelson, The White Stripes, and Warren Zevon. This is the first year of eligibility for Missy Elliott and The White Stripes.</span>
+        </p>
+        <p class="contact">Please contact us per Email for any furter question about R&R Hall of Fame 2023<br><span class="email">r&rhallfame2023@rrhof.org</span></p>
+      </div>
```

Comment:
- [ ] kindly try to use the correct color of the Behance here in the paragraph of the about page and with that the project will be similar to the Behance and nice 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/217926819-6b21581d-4260-41e9-8dba-d8c4bf17f2b4.png)   |   ![image](https://user-images.githubusercontent.com/83360636/217926778-57951356-b9a6-425a-80cb-5dda8e7ed8b4.png) |